source/Cute/Commands/Content/ContentTranslateCommand.cs

```diff
@@ -211,37 +230,74 @@ await ProgressBars.Instance()
                                             }
 
-                                            failedEntryIds[entryId].Add(targetLocaleFieldName);
+                                            return (translatedText, true);
                                         }
-                                        flatEntry[targetLocaleFieldName] = translatedText;
-                                        entryChanged = true;
-                                        taskTranslate.Description = $"{Emoji.Known.Robot} Translating ({symbols} symbols translated)";
+                                        catch (Exception ex)
+                                        {
+                                            _console.WriteAlert($"Error translating text from {defaultLocale.Code} to {targetLocale.Code} using {tService}. Error Message: {ex.Message}");
+                                            return (string.Empty, false);
+                                        }
+                                    });
+
+                                    taskMap.Add(targetLocaleFieldName, translateWrapper());
+
+                                    //if (string.IsNullOrEmpty(translatedText))
+                                    //{
+                                    //    if(!failedEntryIds.ContainsKey(entryId))
+                                    //    {
+                                    //        failedEntryIds[entryId] = new List<string>();
+                                    //    }
+
+                                    //    failedEntryIds[entryId].Add(targetLocaleFieldName);
+                                    //}
+                                    //flatEntry[targetLocaleFieldName] = translatedText;
+                                    //entryChanged = true;
+                                    //taskTranslate.Description = $"{Emoji.Known.Robot} Translating ({symbols} symbols translated)";
+                                }
+                                taskTranslate.Increment(1);
+                            }
+                            
+                            Task.WaitAll(taskMap.Values.ToArray());
+
+                            foreach (var item in taskMap)
+                            {
+                                var (translatedText, success) = item.Value.Result;
+                                if (success)
+                                {
+                                    if(string.IsNullOrEmpty(translatedText))
+                                    {
+                                        if (!failedEntryIds.ContainsKey(entryId))
+                                        {
+                                            failedEntryIds[entryId] = new List<string>();
+                                        }
+
+                                        failedEntryIds[entryId].Add(item.Key);
                                     }
-                                    catch (Exception ex)
+                                    else
```

Comment:
There's a retry loop, but it doesn't actually retry the translation - it just decrements a counter
The retry mechanism should call the translate function again on failure